adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1335,6 +1398,36 @@ abstract class AvroReadGroupGenomicRDD[T <% IndexedRecord: Manifest, U <: Produc
    */
```

Comment:
Marking this as completed --> this was done by @heuermh in https://github.com/bigdatagenomics/adam/pull/1600/commits/79d26d54397672bf5dafafc4c30c928b80a2dc7a.